src/community.c

```diff
@@ -2111,6 +2111,261 @@ int igraph_le_community_to_membership(const igraph_matrix_t *merges,
 
 /********************************************************************/
 
+/**
+ * \ingroup communities
+ * \function igraph_community_fluid_communities
+ * \brief Community detection algorithm based on the simple idea of  
+ * several fluids interacting in a non-homogeneous environment
+ * (the graph topology), expanding and contracting based on their
+ * interaction and density. 
+ *
+ * This function implements the community detection method described in:
+ * Parés F, Gasulla DG, et. al. (2018) Fluid Communities: A Competitive,
+ * Scalable and Diverse Community Detection Algorithm. In: Complex Networks
+ * & Their Applications VI: Proceedings of Complex Networks 2017 (The Sixth 
```

Comment:
The `&` should be replaced by `&amp;`, otherwise making the docs throws an error I think.